backend/src/test/java/ca/bc/gov/hlth/hnweb/controller/GroupMemberControllerTest.java

```diff
@@ -83,6 +86,40 @@ public void testUpdateNumberAndDept_invalidRequest() {
         assertAffectedPartyCount(AffectedPartyDirection.INBOUND, 2);
         assertAffectedPartyCount(AffectedPartyDirection.OUTBOUND, 0);
     }
+    
+    @Test
+    public void testUpdateNumberAndDept_deleteGroupMember_Dept() throws InterruptedException {
```

Comment:
Underscores in test method names should only be used to separate the main part of the test name from the "case" being tested. Just use deleteGroupMemberAndDept or something like that.